src/fsharp/FSharp.Build-proto/FSharp.Build-proto.fsproj

```diff
@@ -47,11 +47,11 @@
     <Reference Include="$(FSCoreLKGPath)" />
     <Reference Include="System" />
     <Reference Include="System.Numerics" />
-    <Reference Include="Microsoft.Build.Framework, Version=12.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" />
-    <Reference Include="Microsoft.Build.Engine, Version=12.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" />
-    <Reference Include="Microsoft.Build, Version=12.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" />
-    <Reference Include="Microsoft.Build.Utilities.v12.0" />
-    <Reference Include="Microsoft.Build.Tasks.v12.0" />
+    <Reference Include="Microsoft.Build.Framework, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" />
```

Comment:
For example, here is the log from building from a VS2013 command prompt on my machine (have VS2010, VS2012, VS2013, VS2015 installed) https://gist.github.com/dsyme/b3838c25b2285bdff0aa

At this line when building FSHarp.Build-proto.dll the MSBuild v4.0 files are being referenced from the GAC.  Ths doesn't seem right https://gist.github.com/dsyme/b3838c25b2285bdff0aa#file-gistfile1-txt-L464

It also seems to happen later when building FSharp.Build.dll at this line: https://gist.github.com/dsyme/b3838c25b2285bdff0aa#file-gistfile1-txt-L574